Makefile

```diff
@@ -1,23 +1,28 @@
-build:
+.PHONY: html
+html:
 	rm -fr public
 	hugo --theme=golangshow
 
+.PHONY: whitespace
+whitespace:
+	go run tools/strip-whitespace/main.go
+
+.PHONY: build
+build: html whitespace
+	$(info -------------------------------------------------------------------------------------)
+	$(info Yeah! The code is ready for upload, please don't forget to check, commit and push it.)
+	$(info After that, you can run `make upload` to publish the site.)
+
+.PHONY: watch
```

Comment:
We can collect all `.PHONY` targets at the end like
```
.PHONY: watch init …
```

However, I only see a benefit in them if we have make targets sharing names with file or directory names. We don't have a `watch` directory, for example, so why we need a phony target at all?